Example/Info.plist

```diff
@@ -15,13 +15,18 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.3</string>
+	<string>0.3.8</string>
```

Comment:
This needs to be 1.0.0. Entirely switching RxSwift version is a major change.

Reply:
ok